app/assets/stylesheets/application.css

```diff
@@ -7,6 +7,7 @@ If you need to create new partials, please import them in partials.scss.
 *= require jquery-ui
 *= require bootstrap-tokenfield
 *= require bootstrap-datepicker3
+*= require fullcalendar
```

Comment:
You have chosen a different order for the CSS asset and the JS asset. Maybe, you could arrange them similar? (In the JS asset, you added your lines between query and the bootstrap stuff, where you added these here after the bootstrap stuff). Of course, this is not necessary.